index.md

```diff
@@ -1,253 +1,95 @@
-  <div class="repo-box">
-    <div class="repo-header">
-      <div class="repo-header-right">
-        <a href="https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/client" alt="GitHub URL">
-          <i class="fa fa-github"></i>
-        </a>
-      </div>
-      @ethereumjs/client
-    </div>
-    <div class="repo-description">
-      <ul>
-        <li>Full sync support</li>
-        <li>Engine API</li>
-        <li>JSON-RPC via HTTP or Websocket</li>
-        <li>Syncs small and mid-size test networks</li>
-        <li>Used for protocol development (EIP-4844)</li>
-      </ul>
-    </div>
-</div>
-</div>
-
-<div class="separator"></div>
-
-<div class="repo-group">
-  <h3><i class="fa fa-cogs"></i> Ultralight</h3>
-  <p>Portal Network client implementation in TypeScript (in Development).</p>
+  <h3>Pectra-Ready Protocol Stack</h3>
+  <p>We provide an up-to-date Ethereum Layer 1 protocol implementation for the TypeScript
+  and browser community. This includes a Pectra-ready EVM but also other secure and
+  browser-ready libraries for dApps, tools and websites. Main code entrypoint here is the EthereumJS 
+  <a href="https://github.com/ethereumjs/ethereumjs-monorepo">monorepo</a>.
+  </p>
 </div>
 
 <div class="repo-group">
   <div class="repo-box">
-    <div class="repo-header">
-      <div class="repo-header-right">
-        <a href="https://github.com/ethereumjs/ultralight" alt="GitHub URL">
-          <i class="fa fa-github"></i>
-        </a>
-      </div>
-      Ultralight
-    </div>
-    <div class="repo-description">
-      <ul>
-        <li>History Network Support</li>
-        <li>State Network (in development)</li>
-        <li>Beacon Chain Network (in development)</li>
-      </ul>
-    </div>
-</div>
+    <img src="/assets/images/evm_example.png" alt="TypeScript EVM - Code Example">
+  </div>
 </div>
 
 <div class="separator"></div>
 
 <div class="repo-group">
-  <h3><i class="fa fa-cube"></i> Blockchain</h3>
-  <p>Core building blocks for an Ethereum blockchain including a transaction library supporting all existing 
-  transaction types.</p>
+  <h3>Protocol R&D</h3>
+  <p>We do various early-on EIP code integrations, see e.g. <a href="https://github.com/ethereumjs/ethereumjs-monorepo/pull/3976">PeerDAS</a>, <a href="https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/binarytree">Binary Tree</a> or native <a href="https://github.com/ethereumjs/ethereumjs-monorepo/pull/3849">SSZ</a> as examples. Along the way we engage in research calls and help with spec refinements on the EIP GitHub like <a href="https://github.com/ethereum/EIPs/pull/9460">here</a> and do coverage work and contribute test cases to the execution-spec-tests GitHub like <a href="https://github.com/ethereum/execution-spec-tests/pull/1371">here</a>. Our EthereumJS <a href="https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/client" alt="GitHub URL">research client</a> is an important enabler for this work.
```

Comment:
Can we rephrase "native SSZ" to something like "ssz-ification"?  It reads like we've built our own ssz library when in fact we're using the `noble` one.

Reply:
👍

Have updated, will push soon.